alf/utils/plot_tb_curves.py

```diff
@@ -229,31 +249,99 @@ def y_label(self):
         return "Success Rate"
 
 
+class MeanCurveGroupReader(object):
+    r"""Group several ``MeanCurveReader`` results. A ``MeanCurveGroupReader`` is
+    suitable for one method on multiple tasks, each task with multiple runs.
+    To aggregate across tasks, each task must be provided with a performance
+    range :math:`(y_0, y_1)` that will be used to normalize performance for that
+    task as :math:`\frac{y - y_0}{y_1 - y_0}`.
+    """
```

Comment:
perhaps also include an example of using this somewhere.

Reply:
updated the example